### PR TITLE
Google TTS

### PR DESCRIPTION
**Pull request by Fr6jDJF**
Google TTS

### CODE REVIEW

File: cogs/audio.py [hunk 1]
```diff
@@ -263,6 +271,63 @@ def __init__(self, bot):
         else:
             await self.bot.say("There are no valid sound effects in the `data/audio/sfx` folder.")
 
+    @commands.command(pass_context=True, no_pm=True)
+    async def tts(self, ctx, lang: str, *, text: str="Please try again."):
+        """Convert your text to speech (Google TTS).
+        """
+        
+        text = self.chop(ctx, text)
```
Comment (irdumbs) [inline cogs/audio.py:279]: `!tts en my string`
text is "my string"
~~from the looks of it, chop will return: "ring"~~
ah nvmd.
but is text ever None?
#chop `if message == None:`

**Comment by irdumbs** [comment]
doesn't look like it accounts for resuming audio while a tts is playing

File: cogs/audio.py [hunk 2]
```diff
@@ -923,6 +985,20 @@ def is_playlist_name_valid(self, name):
             result = {"title": "A song "}
         return result
 
+    def chop(self, ctx, stuff:str):
+        message = stuff
+        if message == None:
+            message = ctx.message.author.name
+        else:
+            message = ctx.message.clean_content
+            message = message[(len(ctx.prefix)+len(ctx.command.name))+1:]
```
Comment (irdumbs) [inline cogs/audio.py:994]: I would be careful of using this.
I believe it bugs out if a prefix is a mention.
I recommend just going through `stuff` and converting the mentions in that.

**Comment by irdumbs** [comment]
please PR this to cog_changes

**Comment by tekulvw** [comment]
I'm seconding that. Closing in anticipation of a PR to `cog-changes`.

**Pull request closed without merging**